waiter/src/waiter/process_request.clj

```diff
@@ -544,40 +526,49 @@
                         ; request-state-chan should be explicitly closed after the request finishes processing
                         request-state-chan (async/tap control-mult (au/latest-chan) false)
                         queue-timeout-ms (:queue-timeout-ms instance-request-properties)
-                        instance (fa/<? (prepare-instance instance-rpc-chan service-id reason-map start-new-service-fn request-state-chan
-                                                          queue-timeout-ms reservation-status-promise metric-group
-                                                          add-debug-header-into-response!))]
+                        instance-timer (metrics/service-timer service-id "get-available-instance")
+                        timed-instance (metrics/with-timer
+                                         instance-timer
```

Comment:
We can inline `instance-timer` and avoid introducing a new variable

Reply:
Done